include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -1910,7 +1913,9 @@ class FEEvaluation : public FEEvaluationAccess<dim,n_components_,Number>
   static const unsigned int dimension     = dim;
   static const unsigned int n_components  = n_components_;
   static const unsigned int static_n_q_points    = Utilities::fixed_int_power<n_q_points_1d,dim>::value;
-  static const unsigned int tensor_dofs_per_cell = Utilities::fixed_int_power<fe_degree+1,dim>::value;
+  static const unsigned int static_dofs_per_component = Utilities::fixed_int_power<fe_degree+1,dim>::value;
+  static const unsigned int tensor_dofs_per_cell = n_components *static_dofs_per_component;
```

Comment:
maybe add space after `*`?

Reply:
That was `astyle`, no idea why it triggers here...

Reply:
ah, ok then.